### ISSUE TITLE
ruby parser errors

### ISSUE THREAD

**Issue by chrishough** [sds/slim-lint#23]
I started getting this error in my application and I am concerned it is related to ruby 2.3.0
```
-->> slim-lint /Users/chrishough/Studio/Code-Consulting/bilingualhire/app/**/*.slim --config=.slimlint.yml
warning: parser/current is loading parser/ruby22, which recognizes
warning: 2.2.x-compliant syntax, but you are running 2.3.0.
warning: please see https://github.com/whitequark/parser#compatibility-with-ruby-mri.
```

**Comment by sds** [comment]
Hey @chrishough,

Those are warnings printed by the `parser` gem, and completely outside the control of `slim-lint`.

You might want to try upgrading to the latest version of the `parser` gem: `bundle update parser` (if you're managing your dependencies via Bundler).

**Comment by chrishough** [comment]
Np, I already tried thats. Seems like rubocop has the same issue.

**Comment by chrishough** [comment]
In case anyone wants these shut off here is a rake task work around:

```ruby
desc 'Run slim-lint which is a ruby scss analyzer'
      task :slimlint do
        msg('Running slim-lint to analyze our slim code')
        shell("RUBYOPT=W0 slim-lint #{Rails.root}/app/**/*.slim --config=.slimlint.yml ")
      end
```
